doc/btp_a2dp.txt

```diff
@@ -0,0 +1,32 @@
+A2DP Service (ID 0x23)
+=====================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read Supported Commands command/response
+
+		Controller Index:	<controller id>
+		Command parameters:	<none>
+		Response parameters:	<supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+Events:
+	Opcode 0x80 - Dummy event
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status  (1 octet)
+
+        This dummy event indicates something that happens asynchronously
+        e.g. IUT finished discovery, a notification arrived, etc.
```

Comment:
Is there a reason that only that one command is described in doc?
Wouldn't it be better to describe all new commands and events?